File: market/python/24HoursVolume.py
import requests
import json

# set the url
url = "https://api.deversifi.com/v1/pub/24HoursVolume/{year}/{month}/{day}"

# set the parameters
url = url.format(year=2020, month=1, day=1)

# standard headers
headers = {
    "Accept": "application/json",
    "Content-Type": "application/json"
}

# execute the request and load into dictionary
response = json.loads(requests.request(
    "GET", url, headers=headers).text)

# print the response from dictionary
print(json.dumps(response, indent=2))

File: market/js/24HoursVolume.js
var request = require("request");
// set parameters
const year = "2020";
const month = "01";
const day = "01";
// set url
const url = "https://api.rhino.fi/v1/pub/24HoursVolume/" + year + "/" + month + "/" + day;
// execute request and print result
request(url, function(error, response, body) {
    console.log(JSON.parse(body));
});